generators/client/templates/angular/webpack/utils.js.ejs

```diff
@@ -28,3 +31,12 @@ function root(args) {
   args = Array.prototype.slice.call(arguments, 0);
   return path.join.apply(path, [_root].concat(args));
 }
+
+function addAliasesFromTypescriptConfig(initialAliases) {
```

Comment:
Can we instead write like below to separate map logic from reduction?
```
function mapTypescriptAliasToWebpackAlias(alias = {}) {
  const webpackAliases = { ...alias };
  Object.entries(tsconfig.compilerOptions.paths)
    .map(([key, value]) => {
      const regexToReplace = /\/\*$/;
      const aliasKey = key.replace(regexToReplace, '');
      const aliasValue = value.length ? value[0].replace(regexToReplace, '') : value.replace(regexToReplace, '');
      return [aliasKey, root(aliasValue)];
    })
    .reduce((aliases, [key, value]) => (aliases[key] = value), webpackAliases);
  return webpackAliases;
}
```

Reply:
I used ideas from this snippet and made code more readable. Thanks for the idea!